.eslintrc.cjs

```diff
@@ -75,5 +76,42 @@ module.exports = {
 
 		// These on-by-default rules don't work well for this repo and we like them off.
 		"no-inner-declarations": "off",
+
+		// JSDoc rules
```

Comment:
[Request] A lot of these are enabled in `"extends": ["plugin:jsdoc/recommended-error"]`. Is there a reason not to extend from that, then configure rules from that basic template?

I generally prefer extending from recommended rulesets so we can defer to the plugin author's expertise on what's best to use. Especially as they add to the recommended list over time.

Reply:
Yeah, it probably best to make this change.
The main reason I didn't initially do this is because there are a lot of additional rules I want to enable, as well as bunch that we want to turn off in a TypeScript environment.